File: emulators/aqemu/files/patch-src_System__Info.cpp
--- src/System_Info.cpp.orig	2017-12-31 00:45:58 UTC
+++ src/System_Info.cpp
@@ -21,6 +21,7 @@
 **
 ****************************************************************************/
 
+#include <unistd.h>
 #include <QSettings>
 #include <QRegExp>
 #include <QProcess>

File: devel/fnm/files/patch-src_system__info.rs
--- src/system_info.rs.orig	2023-01-06 17:40:16 UTC
+++ src/system_info.rs
@@ -8,6 +8,11 @@ pub fn platform_name() -> &'static str {
     "linux"
 }
 
+#[cfg(target_os = "freebsd")]
+pub fn platform_name() -> &'static str {
+    "freebsd"
+}
+
 #[cfg(all(
     target_pointer_width = "32",
     any(target_arch = "arm", target_arch = "aarch64")
